### PR TITLE
add ut for pkg/version/version.go

### PR DESCRIPTION
**Pull request by my-git9**
**What type of PR is this?**
add ut for pkg/version/version.go

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by jwcesign** [comment]
Thanks for contribution

File: pkg/version/version_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			info := Info{
+				GitVersion:   tt.fields.GitVersion,
+				GitCommit:    tt.fields.GitCommit,
+				GitTreeState: tt.fields.GitTreeState,
+				BuildDate:    tt.fields.BuildDate,
+				GoVersion:    tt.fields.GoVersion,
+				Compiler:     tt.fields.Compiler,
+				Platform:     tt.fields.Platform,
+			}
+			if got := info.String(); got != tt.want {
```
Comment (jwcesign) [inline pkg/version/version_test.go:73]: For string function, I think one case is enough. BTW, would u mind helping add ut for `Get` function.

**Comment by XiShanYongYe-Chang** [comment]
Hi @my-git9, thanks a lot! You can pick the related task as #2579 said.

**Comment by my-git9** [comment]
> Hi @my-git9, thanks a lot! You can pick the related task as #2579 said.

OK~~

**Comment by my-git9** [comment]
> I made some changes on your patch, please reference:
> 
> ```go
> func TestInfo_String(t *testing.T) {
> 	tests := []struct {
> 		name string
> 		info Info
> 		want string
> 	}{
> 		{
> 			name: "test1",
> 			info: Info{
> 				GitVersion:   "1.3.0",
> 				GitCommit:    "da070e68f3318410c8c70ed8186a2bc4736dacbd",
> 				GitTreeState: "clean",
> 				BuildDate:    "2022-08-31T13:09:22Z",
> 				GoVersion:    "go1.18.3",
> 				Compiler:     "gc",
> 				Platform:     "linux/amd64",
> 			},
> 			want: `version.Info{GitVersion:"1.3.0", GitCommit:"da070e68f3318410c8c70ed8186a2bc4736dacbd", GitTreeState:"clean", BuildDate:"2022-08-31T13:09:22Z", GoVersion:"go1.18.3", Compiler:"gc", Platform:"linux/amd64"}`,
> 		},
> 	}
> 	for _, tt := range tests {
> 		t.Run(tt.name, func(t *testing.T) {
> 			if got := tt.info.String(); got != tt.want {
> 				t.Errorf("Info.String() = %v, want %v", got, tt.want)
> 			}
> 		})
> 	}
> }
> ```

Thanks a lot!

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**